container/lxd/manager.go

```diff
@@ -145,7 +152,13 @@ func (m *containerManager) getContainerSpec(
 		return ContainerSpec{}, errors.Trace(err)
 	}
 
+	// Lock around finding an image.
+	// The provisioner works concurrently to create containers.
+	// If an image needs to be copied from a remote, we don't many goroutines
```

Comment:
'we don't want many ... doing it at the same time'

Reply:
I'll get it in a drive-by next time :)

Reply:
Will get it in a drive-by next time :)